Спринт №18

Keithcutt opened the pull request:
1. Создан фрагмент для плейлистов.
2. Создан фрагмент для избранных треков.
3. Реализован экран "Медиатека".
4. Созданы вьюмодели для каждого из созданных фрагментов и внедрены в качестве зависимостей.

Keithcutt committed:
1. Создан фрагмент для плейлистов.
2. Создан фрагмент для избранных треков.
3. Реализован экран "Медиатека".
4. Созданы вьюмодели для каждого из созданных фрагментов и внедрены в качестве зависимостей.

Keithcutt committed:
Небольшие изменения в MediaActivity.

ArturNurtdinov commented on app/src/main/res/layout/activity_media.xml:
```diff
@@ -1,9 +1,44 @@
+            style="@style/HeaderBackButton"
+            android:layout_width="@dimen/dp_48"
+            android:layout_height="wrap_content"
+            app:icon="@drawable/ic_arrow_back" />
```
Кажется, что иконку тоже можно в стиль занести, т.к. везде одна и та же стрелочка используется в шапке экрана

ArturNurtdinov commented on app/src/main/res/layout/fragment_favourites.xml:
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Здесь можно сразу заменить на `ConstraintLayout` аналогично `fragment_playlists`, т.к. в будущем на экране будут и другие элементы

ArturNurtdinov commented on app/src/main/res/layout/fragment_playlists.xml:
```diff
@@ -0,0 +1,50 @@
+        android:layout_gravity="center_horizontal"
+        android:layout_marginTop="@dimen/dp_128"
+        android:text="@string/new_playlist_button"
+        app:cornerRadius="@dimen/dp_52"
```
Размер скругления лучше в стиль занести

Keithcutt committed:
Правки после ревью:
1. Иконка "Назад" перенесена в стиль.
2. Главный вьюгруп в макете fragment_favourites заменен на ConstraintLayout.
3. Размер скругления у кнопок занесен в стиль.
